todo_app/data/session_items.py

```diff
@@ -13,7 +15,9 @@ def get_items():
     Returns:
         list: The list of saved items.
     """
-    return session.get('items', _DEFAULT_ITEMS.copy())
+    existing_items = session.get('items', _DEFAULT_ITEMS.copy())
+    existing_items.sort(key=lambda item : item['status'])
```

Comment:
Minor, but this actually breaks some of your functionality! By sorting here, you affect the order of the items that are received by the `add_item` function when it calls get_items (line 47) which means that it's assumption that the new id is ((the final id in the list) plus one) is now broken.

So e.g.:
* Starting with the default list, add a new item
* Make that new item Completed so that it is at the start of the list
* Add another new item
* You'll find that both your new items have the same id, which means modifying/removing either affects both!

To fix this, we could either:
* Do the sorting within the index route
* Change the `add_item` behaviour to find the largest id and then add one, rather than using the last id plus one